frontend/src/assets/stylesheets/components/post.sass

```diff
@@ -6,18 +11,21 @@
   padding: 32px
   position: relative
 
+  &:hover
+    +transition(background-color)
+    background-color: darken($color-black, 1%)
```

Comment:
not sure if darken/lighten is proper way of describing color, as far as i know it is very inconsistent